app/services/websites/create.rb

```diff
@@ -14,7 +14,15 @@ def initialize(params, user)
     end
 
     def call
-      if params[:website][:url].to_s.include? '.'
+      already_exist = Website.find_by(url: params[:website][:url])
+      if already_exist && params[:website][:collection_id]
+        website = CollectionWebsite.where('collection_id = ?', params[:website][:collection_id])
+                                   .where('website_id = ?', already_exist.id)
+                                   .first
+        if website.nil?
+          CollectionWebsite.create!(collection_id: params[:website][:collection_id], website_id: already_exist.id)
+        end
```

Comment:
@Nimmyjv Are you replacing just the `if website.nil?` block or the all the content inside the if condition on line 18? What I meant was that you should replace lines 19-24 with the above code.